Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:22.04
+
+# The dalai server runs on port 3000
+EXPOSE 3000
+
+# Install dependencies
+RUN apt-get update \
+    && apt-get install -y software-properties-common curl
+
+# Add deadsnakes PPA to get Python 3.10
+RUN add-apt-repository ppa:deadsnakes/ppa && apt-get update
+
+# Add NodeSource PPA to get Node.js 18.x
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+
+# Install Python 3.10 and Node.js 18.x
+RUN apt-get update \
+    && apt-get install -y python3.10 python3-venv python3-pip nodejs
+
+# Copy the dalai source code into the container
+WORKDIR /dalai
+
+COPY . .
+
+# Install dalai and its dependencies
+RUN npm install
+
+# Install a model
+RUN npx dalai alpaca install 7B
```

Comment:
This should be using either build time or runtime vars so the user can pick the models they want.